src/main/java/org/jabref/model/groups/TexGroup.java

```diff
@@ -32,6 +30,16 @@
     private final MetaData metaData;
     private String user;
 
+    public TexGroup(String name, GroupHierarchyType context, Path filePath, AuxParser auxParser, FileUpdateMonitor fileMonitor, MetaData metaData, String user) throws IOException {
+        super(name, context);
```

Comment:
Now the code in the constructor below can be replaced by `this(..., System.getProperty("user.name") + '-' + InetAddress.getLocalHost().getHostName())`.
Everytime you copy code, it is a sign that you should improve the code structure instead. Code duplication is a huge burden in the long term since it makes it hard to maintain the code. For example, imagine that somebody changes the line ` this.filePath = expandPath(Objects.requireNonNull(filePath)); ` in one constructor but forgets to change the same line in the other constructor....

Reply:
You are right and that is something I overlooked. Thank you for the feedback. I will commit all the requested changes soon.